src/printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print('x is %(a)d, y is %(b).2f, z is %(c)s' % {'a': x, 'b': y, 'c': z})
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is {a}, y is {b}, z is {c}'.format(
```

Comment:
You can also output a rounded decimal with the second and third print statements. Check out this link for the second one https://mkaz.blog/code/python-string-format-cookbook/ and this link for the third https://stackoverflow.com/questions/45310254/fixed-digits-after-decimal-with-f-strings